diff --git a/src/headers/config.h b/src/headers/config.h
index 3a731ad..12f2961 100644
--- a/src/headers/config.h
+++ b/src/headers/config.h
@@ -48,7 +48,7 @@
 
 #define CFG_GYRO_SENSITIVITY  (pow(2, -9) * 1.45)
 #define CFG_GYRO_SENSITIVITY_X  (CFG_GYRO_SENSITIVITY * 1)
-#define CFG_GYRO_SENSITIVITY_Y  (CFG_GYRO_SENSITIVITY * 1)
+#define CFG_GYRO_SENSITIVITY_Y  (CFG_GYRO_SENSITIVITY * 0.5)
 #define CFG_GYRO_SENSITIVITY_Z  (CFG_GYRO_SENSITIVITY * 1)
 
 #define CFG_MOUSE_WHEEL_DEBOUNCE 1000
